src/java/fr/paris/lutece/plugins/limitconnectedusers/service/filter/LimitConnectedUsersFilter.java

```diff
@@ -131,12 +131,18 @@ public void doFilter( ServletRequest request, ServletResponse response, FilterCh
 
             if ( sessionsActives != null )
             {
-                if ( !sessionsActives.contains( session.getId(  ) ) &&
-                        ( sessionsActives.size(  ) < _nMaxConnectedUsers ) )
+                if ( !sessionsActives.contains( session.getId(  ) ) 
+                        && _nMaxConnectedUsers < 0 )

```

Comment:
The new behavior of nMaxConnectedUsers < 0 needs to be documented and its relation to bActivate
so you want to have the following ?
- bActivate == false => no limit, no counts on the dashboard
- bActivate == true, nMaxConnectedUsers < 0 => no limit, but session count on the dashboard

Reply:
that was the idea.
"Activate" activates the filter 
"MaxConnectedUsers" activates the limit